addons/account/models/account_move.py

```diff
@@ -2997,18 +2997,21 @@ def _post(self, soft=True):
 
         # Create the analytic lines in batch is faster as it leads to less cache invalidation.
         to_post.mapped('line_ids').create_analytic_lines()
-        to_post.write({
-            'state': 'posted',
-            'posted_before': True,
-        })
 
         for move in to_post:
             # Fix inconsistencies that may occure if the OCR has been editing the invoice at the same time of a user. We force the
             # partner on the lines to be the same as the one on the move, because that's the only one the user can see/edit.
-            wrong_lines = move.is_invoice() and move.line_ids.filtered(lambda aml: aml.partner_id != move.commercial_partner_id)
+            wrong_lines = move.is_invoice() and move.line_ids.filtered(lambda aml: aml.partner_id != move.commercial_partner_id
+                                                                                   and aml.display_type not in ['line_section', 'line_note'])
```

Comment:
and not aml.display_type
```suggestion
                                                                                   and not aml.display_type
```

Reply:
ha yes, it's even better :sweat_smile:

Reply:
Thanks @jco-odoo

Reply:
I'm gonna put it back in master anyways :shrug: